File: Algorithms/SelectionSort.py
# Author : Afridi
# 29-11-2018


def Selection_sort(L):
  n = len(L)
  for i in range(n-1):
    index_min = i
    for j in range(i,n):
      if L[j] < L[index_min]:
        index_min = j
    L[i],L[index_min]=L[index_min],L[i]

L = [6,2,1,0,4]
Selection_sort(L)
print('Sorted list',L)   # Sorted list [0,1,2,4,6]

File: Js/Algorithms/SelectionSort.js
// Author : Afridi

function Selection_sort(L){
	var n = L.length;
	for(i=0 ; i<n-1 ; i++){
  	    index=i
		for(j=i ; j<n ; j++){
			if (L[j] < L[index]){
			[L[j],L[index]]=[L[index],L[j]]
			}
		}
	}
}

var L = [0,6,2,1,4]
Selection_sort(L)
console.log('The sorted array',L)
